### DISCUSSION TITLE
Cortx Playground

### DISCUSSION THREAD

**Discussion by stx-hseow** [Seagate/cortx#1468] [Ideas]
Are there any plans to open up a public Cortx "playground" with limited storage (or cleaned up daily/weekly)?

For example, Minio provides a [Play](https://play.min.io/) URL for users to test their apps or integrations. Without the need/hassle to start a K8 or download 2GB OVA (+setup) before they can try it out?

**Comment by novium258** [comment]
We have looked into Cloudshare to provide this, but their usage fees are incredibly steep (basically idle is about $300/month, approximately 25 users logging in  and maybe ~5 more seriously exploring it is about $1500-$2000/month on top of the $13k annual subscription). But I would love to make this happen!

**Reply by stx-hseow** [reply]
Cloudshare is great if you want to provide server access for the users to explore and tinker with. But the playground is sufficient with just the Browser/API access without being able to access (SSH) into the server. That means you could setup Cortx on any kind of server, add a scheduler (to clean up storage regularly) before exposing to the world (EDIT: Which means you only need 1 server).

I am not sure how powerful is the **Play** server provided by Minio, but I believe being able to serve ~15 users simultaneously should be more than enough as a start.

**Comment by mukul-seagate11** [comment]
We are also offering OVA to the community which is a ready VM appliance with CORTX setup so you can evaluate & follow the instructions in https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-5/CORTX_on_Open_Virtual_Appliance_PI-5.rst

**Reply by stx-hseow** [reply]
Yep, and the setup is much easier now when compared to previous versions.

I just think having a "Playground" would encourage more potential users, especially for those with low network bandwidth, unfamiliar with VM setups or having low performance rig.
